### DISCUSSION TITLE
Can you publish a new version soon ?

### DISCUSSION THREAD

**Discussion by AmjadHD** [github-linguist/linguist#6343] [General]
See https://github.com/nim-lang/NimLime/pull/166

**Comment by lildude** [comment]
No, not for a while, unless something that breaks things substantially comes up as the last release was only 2 weeks ago.

**Reply by AmjadHD** [reply]
Nim syntax highlighting is substantially broken.

**Reply by AmjadHD** [reply]
> the last release was only 2 weeks ago

https://github.com/github/linguist/releases/tag/v7.24.1 was released less than two weeks after v7.24.0.

**Reply by lildude** [reply]
> Nim syntax highlighting is substantially broken. 

How? Show me. I don't know Nim at all but from a quick search, many files are highlighting just fine. Please explain to a non-Nim user how the syntax highlighting is substantially broken as I'm not seeing it. The issue you referenced only mentions a certain scenario which doesn't seem to affect all files.

In the case of Svelte which needed the patch release, it was very clear it affected all files. I'm not seeing this with Nim files.

**Reply by AmjadHD** [reply]
Any file containing this pattern is affected ` \{\. *\w+(?:\[\w+\])? *:`.
This [search](https://github.com/search?q=language%3ANim+path%3A*.nim++%2F%5C%7B%5C.+*%5Cw%2B%28%3F%3A%5C%5B%5Cw%2B%5C%5D%29%3F+*%3A%2F&type=code) shows 24.6k from 133k files (18.5%).
